spinn_front_end_common/utilities/constants.py

```diff
@@ -21,12 +21,14 @@
 SPIKE_INJECTOR_CORE_APPLICATION_ID = 0xAC9
 
 BITS_PER_WORD = 32.0
+BYTES_PER_WORS = 4
```

Comment:
Should be "BYTES_PER_WORDS" (and similarly on sPyNNaker branch).  Also, either both of these constants should be floats or both of these constants should be integers; probably the latter?

Reply:
if theres multipl;e of these. then thats a failure on its own. (there should only be a version of this in FEC. no point having the same constants elsewhere.

on floats vs ints. there was a reason to make the bits a float. i think its to do with a numpy expectation. id need to change it and review. will do so after timing

Reply:
ok having investigated. 1. theres 2 params for the same thing. BYTES_PER_WORDS and Bytes_To_WORD_MULTIPLIER, so reduced them. But the issue with flaots vs ints is as follows:
1. the BitsToWord is used to determine bitfield size mostly. and that maths needs a math.ceil of a float, so this removes the casting from int to float all the time. 
2. the BYtes_per_word is used often to find size of a memory region. which requires a int, so by keeping it a int, you avoid more casts.

so summary theyre different types to avoid castings more often for each use case

Reply:
OK, that's fine - thanks.